vskernels/kernels/various.py

```diff
@@ -104,10 +104,7 @@ class EwaBicubic(Placebo):
     _kernel = 'ewa_robidoux'
 
     def __init__(self, b: float = 0.0, c: float = 0.5, radius: int | None = None, **kwargs: Any) -> None:
-        radius = kwargs.pop('taps', radius)
-
-        if radius is None:
-            radius = 1 if (b, c) == (0, 0) else 2
+        radius = self._kernel_size(kwargs.pop('taps', radius), b, c)
```

Comment:
Moved this to the inherited Placebo class because there's multiple Bicubic presets where this may be relevant.